### ISSUE TITLE
HyperlinkButton's UIA properties are announced in an unexpected order

### ISSUE THREAD

**Issue by YuliKl** [microsoft/microsoft-ui-xaml#2987]
**Describe the bug**

The order of information in Narrator is expected to be: "Name LocalizedControlType Value..."
So in the below markup, Narrator should announce "More Link" and "More Button". This is true for the Button control but HyperlinkButton gets announced in the opposite order: "Link More" 

```xaml
            <HyperlinkButton
                AutomationProperties.Name="More"            
                Margin="12,0,0,0"
                Padding="0">
                <TextBlock
                    Grid.Column="0"
                    Text="Click"/>
            </HyperlinkButton>

            <Button
                AutomationProperties.Name="More"
                Margin="12,0,0,0"
                Padding="0">
                <TextBlock
                    Grid.Column="0"
                    Text="Click"/>
            </Button>
```

**Additional context**

Issue found through [internal bugs](https://microsoft.visualstudio.com/OS/_workitems/edit/27493045)
